### ISSUE TITLE
[Research]My VA Redirect: Research artifact review & submission

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#43131]
Tasks to get ready for My VA Redirect UAT

- [x] Get PO and PM review on convo guide created in #43128  and research plan created in #43129
- [x] Update docs as needed
- [x] Submit docs to research repo

**Comment by andaleliz** [comment]
Hi @ajakabcin and @ericasauv09 - I have the [research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/my-va-redirect/UAT/research-plan.md) and [convo guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/my-va-redirect/UAT/conversation-guide.md) ready for review! 

Is it possible for at least one of you to review the docs by 1pm tomorrow (6/23)? I know that is a tight turnaround but I'd love to get this to Samara ASAP so we can submit this and kick off recruiting on Friday. 

I'm waiting on some feedback from accessibility on the phrasing for recruiting criteria, so that update will hopefully come before Friday.

Thanks!

**Comment by ericasauv09** [comment]
@andaleliz Everything looks great to me, thank you!

**Comment by Samara-Strauss** [comment]
Hey Liz! Sorry, just seeing this now. I probably won't get a chance to review this until tomorrow, but I'll let you know if I get to them sooner.

**Comment by andaleliz** [comment]
Thanks @Samara-Strauss! If we can submit this to platform today, I think we can kick off recruiting and start sessions next Friday. Otherwise, we may have to push back to the following week, which is probably OK but means we may need to extend into the 2nd week of July.

**Comment by Samara-Strauss** [comment]
## Conversation guide feedback

### 2. Understand expectation and next steps - 5 minutes

> How does that line up with your expectations?
> Is there anything confusing about this?

I think we should be clearer about this. I think this questions could be misconstrued as "does this page line up with your expectations" and "do you find anything confusing on this page" instead of whether landing on My VA met their expectations or caused confusion. Idk if this is helpful, but I think it's OK to state what happened and maybe clarify these questions like:

- You logged in from the homepage, and now you are seeing a page called My VA. How does that line up with your expectations?
- Is there anything confusing about having logged in from the homepage and now seeing a page called My VA?

---

Also, I realize this may come up naturally, but if it does not, I would specifically ask how they would get back to the homepage if they needed/wanted to. We may also want to ask if they know how to get back to their personalized dashboard if they navigate away from it.

I also would be interested in having them go back to the homepage and talking about whether they need the homepage now that they've been taken to their personalized dashboard, and if so, for what?

**Comment by andaleliz** [comment]
Yessss these are great points!  Thanks for the suggestions @Samara-Strauss and @ajakabcin 

>You logged in from the homepage, and now you are seeing a page called My VA. How does that line up with your expectations?
>Is there anything confusing about having logged in from the homepage and now seeing a page called My VA?

I added this to task 2.

>Also, I realize this may come up naturally, but if it does not, I would specifically ask how they would get back to the homepage if they needed/wanted to. We may also want to ask if they know how to get back to their personalized dashboard if they navigate away from it.

>I also would be interested in having them go back to the homepage and talking about whether they need the homepage now that they've been taken to their personalized dashboard, and if so, for what?

>do they understand or recognize the difference between the homepage and My VA? 

I've added another task between 2-3 called "[Understand the homepage and My VA relationship](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/my-va-redirect/UAT/conversation-guide.md#2-understand-expectation-and-next-steps---5-minutes)".  At the end of task two the user will be somewhere else on the site, so I'm asking them to show me how they would get back to My VA, and then how they'd get back to the homepage once they're there.  On the homepage, I'll prompt them to talk about the differences between the two pages, and how the homepage would be used now that they're familiar with My VA. 

I think the conversation will flow pretty naturally and help us get insights into all of these excellent questions!

Oh, and yes, that was a copy-paste carryover in the research plan. It's been removed.

One last update - the Research Ops team is trying to gather feedback from participants about our practice of holding observers in the waiting room until we get consent. I added [a few questions ](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/my-va-redirect/UAT/conversation-guide.md#before-we-end-this-session-i-have-three-additional-questions-for-you-about-the-study-setup-ill-ask-the-observers-to-leave-the-session-at-this-time)provided to me by Platform to the post-task interview.
